### PR TITLE
sing in

### PR DESCRIPTION
**Pull request by Amrmamlook**

### CODE REVIEW

File: sign in/sing up.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<html>
+<head>
+    
+    
+    <title>
```
Comment (Mohammed4mach) [inline sign in/sing up.html]: ### Title tag should not be empty

File: sign in/sing up.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    </title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    
+</head>
+<body>
+    <div class="login_box">
+        <img src="icons8-user-64.png">
```
Comment (Mohammed4mach) [inline sign in/sing up.html]: ### It is recommended to use alt attribute with img tag with descriptive description of the image

File: sign in/sing up.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<html>
```
Comment (Mohammed4mach) [inline sign in/sing up.html]: ### \<\!DOCTYPE html\> is missing
This tag is important. It Provide information to the browser including the type of the document and how the browser will render the content.
[What is the DOCTYPE Declaration in HTML?](https://www.freecodecamp.org/news/what-is-the-doctype-declaration-in-html/#:~:text=The%20HTML%20document%20type%20declaration,way%20by%20different%20web%20browsers.)

File: sign in/sing up.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+</head>
+<body>
+    <div class="login_box">
+        <img src="icons8-user-64.png">
+        <br>
+        <h1> Sign In</h1>
+        <form action="#">
+            <div class="input_box">
+           
+            <p></p>
```
Comment (Mohammed4mach) [inline sign in/sing up.html]: ### \<p\> tag should not be empty
If it is empty, so it is useless. Please delete or fill with meaningful data.
same problem at line 26

File: sign in/sing up.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            
+            <div class="sign_up">
+                New here? <a href="#">Sign Up</a>
+            </div>
+        </form>
+    </div>
+    
+ </body>
+
+</html>
```
Comment (Mohammed4mach) [inline sign in/sing up.html]: ### It is recommended to have a new line character at the end of the file
[Check why..](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

File: sign in/style.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+    padding: 70px 20px;
+}
+h1
+{
+    margin: 0;
+    padding: 15 0 15px;
+    text-align: center;
+    font-size: 30px;
+    font-family:cursive;
+
```
Comment (Mohammed4mach) [inline sign in/style.css:29]: ### It is bad practice to leave empty lines before finishing the selector style
Same problem in lines 36, 43, 44, 45, 62, 63, 72, 83 and 99

File: sign in/style.css [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+    margin-left: 50px;
+
+}
+img{
+    width: 75px;
+    margin-top: -110px;
+    margin-left: 113px;
+}
+
+
```
Comment (Mohammed4mach) [inline sign in/style.css:107]: ### One new line is enough
Line 106 is enough as empty line at the end of the file.
Lines 107, 108 and 109 is not necessary.

**Review by Mohammed4mach — CHANGES_REQUESTED** [review]
Placeholder color of the input make it difficult to read, please change it besides the requested changes with each file.
Also be aware of typos, the file is intended as sign in.html, its name is sing up.html

File: sign in/sing up.html (hunk 1 above)
Comment (Amrmamlook) [reply sign in/sing up.html]: delete the title lol

Comment (Amrmamlook) [reply sign in/sing up.html]: delete it

File: sing in.html [hunk 8]
```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+    
+
```
Comment (Mohammed4mach) [inline sing in.html]: ### Title tag is missing
Please put in a title tag with descriptive title.

[More about importance of title tag](https://moz.com/learn/seo/title-tag#:~:text=Why%20are%20title%20tags%20important,your%20page%20via%20organic%20search.)

Comment (Mohammed4mach) [inline sing in.html]: ### Title tag is missing
Please put in a title tag with descriptive title.

[More about importance of title tag](https://moz.com/learn/seo/title-tag#:~:text=Why%20are%20title%20tags%20important,your%20page%20via%20organic%20search.)

File: sing in.html [hunk 9]
```diff
@@ -0,0 +1,43 @@
+<html>
```
Comment (Mohammed4mach) [inline sing in.html]: ### \<\!DOCTYPE html\> is missing
This tag provide information and rules to browsers to render page content.
Forget it and you will encounter UNPREDICTED rendering.

[More about this tag](https://graybox.co/knowledge/blog/the-importance-of-the-doctype#:~:text=Per%20HTML%20and%20XHTML%20standards,correctly%20or%20validate%20without%20them.)

File: style.css [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+
+}
+img{
+    width: 75px;
+    margin-top: -110px;
+    margin-left: 113px;
+}
+
+
+
```
Comment (Mohammed4mach) [inline style.css]: ### One new line at the end of a file is enough
Delete Lines 101, 102 and 103

Comment (Mohammed4mach) [inline style.css]: ### One new line at the end of a file is enough
Delete Lines 101, 102 and 103

File: sing in.html [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+            <br>
+            <div class="sign_up">
+                New here? <a href="#">Sign Up</a>
+            </div>
+        </form>
+    </div>
+    
+ </body>
+
+</html>
```
Comment (Mohammed4mach) [inline sing in.html]: ### It is recommended to have a new line character at the end of the file
[Check why..](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

File: style.css [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+    padding: 70px 20px;
+}
+h1
+{
+    margin: 0;
+    padding: 15 0 15px;
+    text-align: center;
+    font-size: 30px;
+    font-family:cursive;
+
```
Comment (Mohammed4mach) [inline style.css]: ### It is bad practice to leave empty lines before finishing the selector style

Same problem at lines 37, 38, 39, 56, 57, 66, 78 and 93

**Review by Mohammed4mach — CHANGES_REQUESTED** [review]

**Pull request merged**